### ISSUE TITLE
The field returns the default value when empty.

### ISSUE THREAD

**Issue by ReVoid** [markglibres/mbran-umbraco-textbox-defaultvalue#1]
I know, it is what package must do, but its cruel for usability:
1. User want to erase field and fill it by new value
2. He trying to erase the last char, but "pooof", now value is default in time he wanted to fill it by new one
3. User forced to not erase last char, enter value he wanted, next erase first char instead

Kind of weird , don't you think? :) 
I think return a default value on focus lost or something like than would be much more intuitive.

May be should add another feature: config field like "Is value fixed" to would extends functionality and make value really fixed, it may be usefull for several cases 

Good package anyway, thanks! :)

**Comment by markglibres** [comment]
@ReVoid , yeah probably repopulate the default value on lost focus..  will add that to my list once I'm back working in Umbraco, probably this month...
